src/components/molecules/ChatMessageBox/ChatMessageBox.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from "react";
+import { useForm } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane } from "@fortawesome/free-solid-svg-icons";
+
+import "./ChatMessageBox.scss";
+
+export interface ChatMessageBoxProps {
+  sendMessage: (text: string) => void;
+}
```

Comment:
Where will `sendMessage` be passed in from? Is it going to be an async function? If so, we probably want to expose the fact that it's a `Promise` or similar in this API here, so that we can then react to it's various states (eg. finished, error, etc) easily.

Reply:
it looks like this one: 
```
  const sendMessage = useCallback(
    (text: string) => {
      if (!venueId || !userId) return;

      const message = buildMessage<VenueChatMessage>({ from: userId, text });

      sendVenueMessage({ venueId, message });
    },
    [venueId, userId]
  );
```

Will left as it is